.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -37,4 +37,4 @@ assignees: ""
 
 ## Extra information
 
-*Optional - Add any extra information which will be usefull for understanding and developing the feature*
+*Optional - Add any useful information to understand and develop the feature*
```

Comment:
Why not change title also ?

Reply:
This was just a simplification of this sentence, to also avoir repeating the title which still seems good as-is. Do you agree with this?

Reply:
Ok for me.